example/lib/example.dart

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: annotate_overrides
```

Comment:
would it be too noisy to put them in?

Reply:
Eh, if we want to copy-paste these into the README.

Reply:
I'd be fine with copying them to the README. Except in cases where doing the "right"/idiomatic thing is very verbose I like for examples to follow good conventions.